src/components/campaign/campaignTable.jsx

```diff
@@ -0,0 +1,219 @@
+import React from 'react'
+import { Table, Column, Cell } from 'fixed-data-table';
+import Moment from 'moment';
+import Promise from 'bluebird';
+
+var SortTypes = {
```

Comment:
No need to change it, but I would favor using `const` or `let` for declaring variables